src/main/webapp/queue/student.html

```diff
@@ -81,7 +81,7 @@
       makeRequest();
     }
 
-    let repeat = setInterval(makeRequest, 5000);
+    let repeat = setInterval(makeRequest, 10000);
```

Comment:
10 seconds seem a bit too long. When I do short polling normally I use a 1 second interval.

Reply:
got it!